src/Api/Model/Shared/Command/UserCommands.php

```diff
@@ -53,7 +53,7 @@ public static function updateUser($params, $website)
         }
 
         UserCommands::assertUniqueIdentity($user, $params['username'], $params['email']);
-        $user->setProperties(UserModel::$ADMIN_ACCESSIBLE, $params);
+        $user->setProperties(UserModel::ADMIN_ACCESSIBLE, $params);
 
```

Comment:
Need to revert this one, its been merged the wrong way. I plan on removing this commit and will force push.